app/views/catarse_bootstrap/projects/_dashboard_posts.html.slim

```diff
@@ -19,14 +19,15 @@
 
               .card.card-message.u-radius.u-marginbottom-40
                 .fontsize-small= t('.warning')
-              .w-row
-                .w-col.w-col-4.w-sub-col.u-marginbottom-20
+
+              .w-row.u-marginbottom-20
+                .w-col.w-col-2
+                .w-col.w-col-5.w-sub-col
```

Comment:
Again, here we should use w-col-push-2 to the w-col-5 and remove the empty w-col-2 div.